src/Dapr.Extensions.Configuration/DaprSecretDescriptor.cs

```diff
@@ -31,20 +31,51 @@ public class DaprSecretDescriptor
         public IReadOnlyDictionary<string, string> Metadata { get; }
 
         /// <summary>
-        /// Secret Descriptor Construcutor
+        /// This flag indicates if this field's existence should trigger an exception. Setting it to "false"
+        /// will suppress the exception whereas setting it to "true" will not suppress it.
+        /// </summary>
+        public bool IsRequired { get; }
+
+        /// <summary>
+        /// SecretKey value is mapping value to Vault Name. If The application's Secret Name and Secret in the
```

Comment:
Nit: the description could be more agnostic as to the backend service.  That is, it may be something other than Azure Key Vault.  The description for `SecretName` could be improved to better highlight the differences between the two.

Reply:
I've updated the descriptions on the properties in the descriptor to be more reflective of their usage. Please let me know if this now makes sense.